src/observableApiAuth.ts

```diff
@@ -149,3 +152,19 @@ export async function whoami(effects = defaultEffects) {
 export function formatUser(user: {name?: string; login: string}): string {
   return user.name ? `${user.name} (@${user.login})` : `@${user.login}`;
 }
+
+export function validWorkspaces(
+  workspaces: GetCurrentUserResponse["workspaces"]
+): GetCurrentUserResponse["workspaces"] {
+  return workspaces.filter((w) => {
+    if (
+      VALID_TIERS.has(w.tier) &&
+      (w.role === "owner" ||
+        w.role === "member" ||
+        (w.role === "guest_member" &&
+          w.projects_info.some((info) => info.project_role === "owner" || info.project_role === "editor")))
+    )
+      return true;
+    return false;
```

Comment:
```suggestion
    return (
      VALID_TIERS.has(w.tier) &&
      (w.role === "owner" ||
        w.role === "member" ||
        (w.role === "guest_member" &&
          w.projects_info.some((info) => info.project_role === "owner" || info.project_role === "editor")))
    )
```

Since this just returns true/false, we can simplify it. (Prettier might not like this as formatted here, sorry)